EIPS/eip-draft-whisper.md

```diff
@@ -72,7 +73,51 @@ This packet is used by Whisper nodes for dynamic adjustment of their individual
 
 This packet contains two objects: integer (0x05) followed by a byte array of arbitrary size.
 
-This packet is used by Whisper nodes for sharing their interest in messages with specific topics. 
+This packet is used by Whisper nodes for sharing their interest in messages with specific topics.
+
+### Whisper Envelope
+
+Envelopes are RLP-encoded structures of the following format:
+
+	[ Version, Expiry, TTL, Topic, AESNonce, Data, EnvNonce ]
+	
+`Version`: up to 4 bytes (currently one byte containing zero). Version indicates encryption method. If Version is higher than current, envelope could not be decrypted, and therefore only forwarded to the peers.
+
+`Expiry`: 4 bytes (UNIX time in seconds).
```

Comment:
actually a decent point. what's important to the receiver is how long the message has left to live, not the time that it was initially meant to be live for.